dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -1,17 +1,126 @@
+# Similar to tempAuth
+# Mandatory if you choose keystone 2 auth
+# objectstorage.swift.keystone2.username=tester
+# Mandatory if you choose keystone 2 auth
+# objectstorage.swift.keystone2.tenantname=test
+
+# ================================ ObjectStorage on Swift with Keystone 3 Auth ==============================
+# Specifying identity information with Keystone 3 Auth
+# Mandatory if you choose keystone 3 auth
+# objectstorage.swift.keystone3.user.name=demo
+# Mandatory if you choose keystone 3 auth
+# objectstorage.swift.keystone3.user.domain=Default
+# Mandatory if you choose keystone 3 auth
+# objectstorage.swift.keystone3.scope.domainid=default
+
+# Specifying project information in KeyStone 3 Auth (similar to tenant in KeyStone 2)
+# Optional
+# objectstorage.swift.keystone3.scope.project.name=test
+# Optional
+# objectstorage.swift.keystone3.scope.project.domainname=Default
+# Optional
+# objectstorage.swift.keystone3.scope.project.domainid=default
+
+# ========================================= ObjectStorage on S3 =============================================
+# Mandatory if you choose aws-s3 storage service, S3 authentication endpoint
+# objectstorage.s3.endPoint=http://james-s3:8000
 
+# Mandatory if you choose aws-s3 storage service, access key id configured in S3
+# objectstorage.s3.accessKeyId=accessKey
+
+# Mandatory if you choose aws-s3 storage service, secret key configured in S3
+# objectstorage.s3.secretKey=secretKey
+
+# ============================================ Blobs Exporting ==============================================
+# Choosing blob exporting mechanism, allowed mechanism are: localFile, linshare
+# LinShare is a file sharing service, will be explained in the below section
+# Optional, default is localFile
 blob.export.implementation=localFile
-blob.export.localFile.directory=file://var/blobExporting
\ No newline at end of file
+
+# ======================================= Local File Blobs Exporting ========================================
+# Optional, directory to store exported blob, directory path follows James file system format
+# default is file://var/blobExporting
+blob.export.localFile.directory=file://var/blobExporting
+
+# ======================================= LinShare File Blobs Exporting ========================================
+# LinShare is a sharing service where you can use james, connects to an existing LinShare server and shares files to
+# other mail addresses as long as those addresses available in LinShare. For example you can deploy James and LinShare
+# sharing the same LDAP repository
+# Mandatory if you choose LinShare, url to connect to LinShare service
+# blob.export.linshare.url=http://linshare:8080
+
+# Mandatory if you choose LinShare, access token to connect to LinShare service. It will be formalized to `Bearer` + a space + access token
+# So, no need to pass `Bearer ` prefix
```

Comment:
extra space

Reply:
did you mean `Bearer `? that is my intention because the linshare access token always is `Bearer` + a space + token

Reply:
Maybe you can explicit it, eg. `("Bearer" + <space>)`